tests/database/image/unit/test_image.py

```diff
@@ -41,13 +47,16 @@ def tearDown(self):
 
```

Comment:
1st one we need a loop as it is a 2D array, need to think about it more.
2nd one need to change it.
3rd there is a cascade rule in db, when we delete project all images under it are also gone in db.